bindings/bindings-linux-aarch64-R4.2.rs

```diff
@@ -348,28 +354,37 @@ pub struct R_allocator {
     _unused: [u8; 0],
 }
 pub type R_allocator_t = R_allocator;
-pub const nchar_type_Bytes: nchar_type = 0;
-pub const nchar_type_Chars: nchar_type = 1;
-pub const nchar_type_Width: nchar_type = 2;
+#[repr(u32)]
 #[doc = "../main/character.c :"]
-pub type nchar_type = u32;
-pub const cetype_t_CE_NATIVE: cetype_t = 0;
-pub const cetype_t_CE_UTF8: cetype_t = 1;
-pub const cetype_t_CE_LATIN1: cetype_t = 2;
-pub const cetype_t_CE_BYTES: cetype_t = 3;
-pub const cetype_t_CE_SYMBOL: cetype_t = 5;
-pub const cetype_t_CE_ANY: cetype_t = 99;
+#[derive(Debug, Copy, Clone, Hash, PartialEq, Eq)]
+pub enum nchar_type {
+    Bytes = 0,
+    Chars = 1,
+    Width = 2,
+}
+#[repr(u32)]
 #[doc = "cetype_t is an identifier reseved by POSIX, but it is\nwell established as public.  Could remap by a #define though"]
-pub type cetype_t = u32;
+#[derive(Debug, Copy, Clone, Hash, PartialEq, Eq)]
+pub enum cetype_t {
+    CE_NATIVE = 0,
+    CE_UTF8 = 1,
+    CE_LATIN1 = 2,
+    CE_BYTES = 3,
+    CE_SYMBOL = 5,
+    CE_ANY = 99,
+}
 #[doc = "Finalization interface"]
 pub type R_CFinalizer_t = ::std::option::Option<unsafe extern "C" fn(arg1: SEXP)>;
 pub type R_pstream_data_t = *mut ::std::os::raw::c_void;
-pub const R_pstream_format_t_R_pstream_any_format: R_pstream_format_t = 0;
-pub const R_pstream_format_t_R_pstream_ascii_format: R_pstream_format_t = 1;
-pub const R_pstream_format_t_R_pstream_binary_format: R_pstream_format_t = 2;
-pub const R_pstream_format_t_R_pstream_xdr_format: R_pstream_format_t = 3;
-pub const R_pstream_format_t_R_pstream_asciihex_format: R_pstream_format_t = 4;
-pub type R_pstream_format_t = u32;
+#[repr(u32)]
+#[derive(Debug, Copy, Clone, Hash, PartialEq, Eq)]
+pub enum R_pstream_format_t {
+    R_pstream_any_format = 0,
```

Comment:
Crazy how there are like 3 enums within one screen and they all have different naming convention. Ideally I'd prefer to drop `R_pstream` prefix here

Reply:
I don't think I can

```c

typedef enum {
    R_pstream_any_format,
    R_pstream_ascii_format,
    R_pstream_binary_format,
    R_pstream_xdr_format,
    R_pstream_asciihex_format
} R_pstream_format_t;
```
They wrote them like that :(